core/src/main/java/org/elasticsearch/index/indexing/IndexSlowLog.java

```diff
@@ -38,7 +34,7 @@
 
 /**
  */
-public class ShardSlowLogIndexingService extends AbstractIndexShardComponent {
+public class IndexSlowLog implements IndexSettingsService.Listener {
```

Comment:
I see what now where this come from - we have IndexSlowLog and SearchSlowLog. To avoid this confusion in the future, can we call this one IndexingSlowLog?